packages/sdk/src/state/zustand.ts

```
@@ -20,6 +20,39 @@ type CreateStore = {
     ): StoreApi<T>;
 };
 
+function deepMerge<T extends SchemaType>(
+    target: ParsedEntity<T>["models"][keyof ParsedEntity<T>["models"]],
+    source: Record<string, any>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)